docs/pages/enterprise/soc2.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: SOC2 compliance for SSH, Kubernetes and Applications
-description: How to configure SSH, Kubernetes and Application access to be SOC2 compliant
-h1: SOC2 Compliance for SSH and Kubernetes and Application Access
+title: SOC2 compliance for SSH, Kubernetes and Databases
```

Comment:
```suggestion
title: SOC2 Compliance for SSH, Kubernetes and Databases
```

Reply:
Should we say "for your Infrastructure" instead of using part of the full list here?